README.md

```diff
@@ -112,6 +112,136 @@ The script has been tested with the KALDI Git hash "7637de77e0a77bf280bef9bf484e
  - Random and deterministic
  - Exclude verbalized punctuation
 
+## How to use this database?
+
+Once you installed this repository and created the sms_wsj database,
+there are a few ways, how you can use this database:
+
+ - Manually read the files from the filesystem (Recommended, when you don't work with python or don't want to use the provided code)
+ - Manually read the json (Not recommended)
+ - Use some helper functions from us to:
+   - Load all desired files from the disk (Recommended for local file systems)
+   - Load only original WSJ utterances and the RIRs and generate the examples on the fly with the help of the json. (Recommended for remote file systems, we mainly use this)
+     - This requires some CPU time. It can be done in a backgroud threadpool, e.g. `lazy_dataset.Dataset.prefetch`, for NN experiments, where the CPU often idles, while the GPU is working.
+     - This allows dynamic mixing of the examples, e.g. creating a nearly infinitely large training dataset.
+
+On the file system you will find files like `.../observation/train_si284/0_4axc0218_01kc020f.wav`.
+Here an explanation, how the path and file name are generated:
+-  `observation`: signal type
+    -  `observation` = `early` + `tail` + `noise`
+    -  `speech_source` convolved with `rirs` = `early` + `tail`
```

Comment:
Is there a `speech_image =` missing?

Reply:
I am not sure I get the meaning of these lines. If you want to say that these are signal types one can work, it is confusing that the key observation is mentioned twice.

Reply:
Good point, I should mention, that this must be calculated